src/scenes/Event/scenes/oracle.js

```diff
@@ -469,7 +467,7 @@ export default class OraclePage extends Component {
     const { locale, messages: localeMessages } = this.props.intl;
     const intl = getIntlProvider(locale, localeMessages);
 
-    if (!this.props.appLocation) this.props.setAppLocation(AppLocation.bet);
+    if (!this.props.appLocation) this.props.store.ui.location = AppLocation.bet;
```

Comment:
shouldnt these check the mobx store location?
`if (!this.props.store.ui.location) ...`

Reply:
yes. will change